spec/system/placements/organisations/users/user_removes_other_users_from_organisation_spec.rb

```diff
@@ -0,0 +1,150 @@
+require "rails_helper"
+
+RSpec.describe "Placements support user removes a user from an organisation", type: :system, service: :placements do
+  describe "schools" do
+    let(:school) { create(:placements_school) }
+    let(:anne) { create(:placements_user, :anne) }
+    let(:mary) { create(:placements_user, :mary) }
+
+    before do
+      [anne, mary].each do |user|
+        create(:membership, user:, organisation: school)
+      end
+    end
+
+    scenario "user is removed from a school" do
+      given_i_am_signed_in_as_mary
+      and_i_visit_the_user_page(anne)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, school)
+      when_i_click_on("Cancel")
+      then_i_return_to_user_page(anne, school)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, school)
+      when_i_click_on("Remove user")
+      then_the_the_user_is_removed_from_the_organisation(anne, school)
+      and_message_is_sent_to_user(anne, school)
+    end
+
+    scenario "I try to remove myself from a school" do
+      given_i_am_signed_in_as_mary
+      when_i_try_to_visit_the_remove_path_form_myself(mary, school)
+      then_i_am_redirected_to_the_index_page(school)
+    end
+  end
+
+  describe "providers" do
+    let(:provider) { create(:placements_provider) }
+    let(:anne) { create(:placements_user, :anne) }
+    let(:mary) { create(:placements_user, :mary) }
+
+    before "message is sent to user" do
+      [anne, mary].each do |user|
+        create(:membership, user:, organisation: provider)
+      end
+    end
+
+    scenario "user is removed from a school" do
+      given_i_am_signed_in_as_mary
+      and_i_visit_the_user_page(anne)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, provider)
+      when_i_click_on("Cancel")
+      then_i_return_to_user_page(anne, provider)
+      when_i_click_on("Remove user")
+      then_i_am_asked_to_confirm(anne, provider)
+      when_i_click_on("Remove user")
+      then_the_the_user_is_removed_from_the_organisation(anne, provider)
+      and_message_is_sent_to_user(anne, provider)
+    end
+
+    scenario "I try to remove myself from a school" do
+      given_i_am_signed_in_as_mary
+      when_i_try_to_visit_the_remove_path_form_myself(mary, provider)
+      then_i_am_redirected_to_the_index_page(provider)
+    end
+  end
+
+  private
+
+  def given_i_am_signed_in_as_mary
+    visit personas_path
+    click_on "Sign In as Mary"
+  end
+
+  def and_message_is_sent_to_user(user, organisation)
+    email = ActionMailer::Base.deliveries.find do |delivery|
+      delivery.to.include?(user.email) && delivery.subject == "You have been removed from #{organisation.name}"
+    end
+
+    expect(email).to_not be_nil
+  end
+
+  def when_i_click_on(text)
+    click_on text
+  end
+
+  def and_i_visit_the_user_page(user)
+    click_on "Users"
+    click_on user.full_name
+  end
+
+  def when_i_try_to_visit_the_remove_path_form_myself(user, organisation)
```

Comment:
Nitpick: This this a typo?

```suggestion
  def when_i_try_to_visit_the_remove_path_for_myself(user, organisation)
```

And actually, now I think about it, is `_myself` actually needed? Looks like it could be used for any user, not only the current user.

```suggestion
  def when_i_try_to_visit_the_remove_path_for(user, organisation)
```